web/src/pages/apd/activities/state-costs/NonPersonnelCost/NonPersonnelCostForm.test.js

```diff
@@ -145,3 +164,25 @@ describe('the ContractorResourceForm component', () => {
     expect(error).toBeInTheDocument();
   });
 });
+
+describe('Selection removes "Administrative operations" for in MMIS apd', () => {
+  beforeEach(() => {
+    mockFlags({ enableMmis: true });
+  });
+
+  afterAll(() => {
+    resetLDMocks();
+  });
+
+  it('shows "Administrative operations" on HITECH APD', async () => {
+    await setup({}, hitechApd);
+
+    expect(screen.getAllByRole('option').length).toBe(7);
+  });
+
+  it('shows "Administrative operations" on HITECH APD', async () => {
```

Comment:
**Typo:** I think you meant to change the name of this test to be for MMIS showing the default list of options